src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -140,7 +147,6 @@ public function configureOptions(OptionsResolver $resolver)
                         $options['em'],
                         $options['class'],
                         $qbParts,
-                        $options['loader'],
```

Comment:
as the loader is now returned from `*Type::getLoader()`, it'll always be the same. I don't think it makes sense to include it in the cache hash anymore.

This was also causing caching to become useless, as the object returned from `getLoader()` always was a new instance, resulting in another hash.